<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>250</height>
     </rect>
    </property>
    <widget class="QLabel" name="Backgroud_Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/background/Tip.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Title_Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>380</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Content_Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>380</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>content</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="OK_Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>125</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed {		
	background-color: rgb(34, 35, 38);
}
QPushButton {
	border:none;
	border-radius:5px;
	
	color: rgb(244, 216, 135);
	background-color: rgb(57, 59, 64);
}</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Tip.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OK_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
